tephra-examples/pom.xml

```diff
@@ -52,7 +64,7 @@
     <dependency>
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-common</artifactId>
-      <version>${hbase12.version}</version>
+      <version>${hbase11.version}</version>
```

Comment:
HBase/Hadoop dependencies can be removed from the parent pom since the child poms will redefine them anyway, right?